src/vcpkg/install.cpp

```diff
@@ -547,8 +547,9 @@ namespace vcpkg
     static constexpr StringLiteral OPTION_PROHIBIT_BACKCOMPAT_FEATURES = "x-prohibit-backcompat-features";
     static constexpr StringLiteral OPTION_ENFORCE_PORT_CHECKS = "enforce-port-checks";
     static constexpr StringLiteral OPTION_ALLOW_UNSUPPORTED_PORT = "allow-unsupported";
+    static constexpr StringLiteral OPTION_NO_PRINT_USAGE = "no-print-usage";
```

Comment:
alternatives:
```
--no-usage
--disable-usage
```

Reply:
I prefer the original `--no-print-usage` over the alternatives. Is the reason for this PR just to reduce verbosity?  

I think that the install command is already bloated with parameters as it is. I'll rather have a `--quiet` (or something akin to `--verbose-level=#`) that suppresses messages in a more comprehensive way than start introducing individual parameters per message type.

Reply:
> Is the reason for this PR just to reduce verbosity?

cmake logs get spammed by cmake usages in manifest mode. I already know the usage since otherwise cmake will fail at configure. Especially for CI runs seeing the usage is kind of pointless. `--quite` is too general. I still want to have general vcpkg output. `--verbose-level=#` would require vcpkg to actually rank the output message consistently between the tool and the cmake script invoke.